data_steward/cdr_cleaner/cleaning_rules/drop_orphaned_survey_conduct_ids.py

```diff
@@ -0,0 +1,144 @@
+"""
+Various cleaning rules will alter data in the observation table, both before
+and as part of de-identification.
+The survey_conduct table needs to remain in sync with the observation table,
+meaning if a survey_conduct.survey_conduct_id exists then at least one
+observation.questionnaire_response_id record with the same value should also exist.
+
+This cleaning rule sandboxes and deletes records from the survey_conduct table
+if the record cannot be joined to the observation table.
+
+Original Issues: DC-2735
+"""
+
+# Python imports
+import logging
+
+# Project imports
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
+from constants.cdr_cleaner.clean_cdr import COMBINED, CONTROLLED_TIER_DEID, CONTROLLED_TIER_DEID_CLEAN, QUERY, RDR, REGISTERED_TIER_DEID, REGISTERED_TIER_DEID_CLEAN
```

Comment:
If you package the import like a tuple, the line will break easily and likely follow the yapf coding style.
```suggestion
from constants.cdr_cleaner.clean_cdr import (COMBINED, CONTROLLED_TIER_DEID, CONTROLLED_TIER_DEID_CLEAN, QUERY, RDR, REGISTERED_TIER_DEID, REGISTERED_TIER_DEID_CLEAN)
```